crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -440,6 +547,117 @@ TEST(ServiceIndicatorTest, AESGCM) {
   ASSERT_EQ(approved, AWSLC_NOT_APPROVED);
 }
 
+TEST(ServiceIndicatorTest, ECDH) {
```

Comment:
Do we need KATs here really? We should be able to generate two ephemeral key pairs and perform an ECDH and run through all curves.

Reply:
Didn't know this was doable at the time, before writing tests for ECDSA. Just updated :)